### PR TITLE
Link to new brief submitted screen from dashboard

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: public/custom.css [hunk 1]
```diff
@@ -153,6 +144,18 @@ form .hint {
 input[type="radio"].uikit-control-input__input:focus + .uikit-control-input__text:before {
   border-radius: 15px;
 }
+.uikit-btn {
```
Comment (matt-sm) [inline public/custom.css]: @marcoalfonso - thoughts?

Comment (maxious) [reply public/custom.css]: This should restore the styling of "buttons wrongly marked up :)" http://uikit.apps.staging.digital.gov.au/packages/buttons/tests/site/ without changing every link in every article.

File: public/custom.css [hunk 2]
```diff
@@ -153,6 +144,18 @@ form .hint {
...
+  color: #fff !important;
+}
+.uikit-btn--tertiary {
+  color: #313131 !important;
```
Comment (marcoalfonso) [inline public/custom.css]: Don't use important!!!  

Just put a class on top and do 
.class .uikit-btn

File: public/custom.css (hunk 1 above)
Comment (marcoalfonso) [reply public/custom.css]: NO IMPORTANT!

File: public/custom.css [hunk 3]
```diff
@@ -153,6 +144,18 @@ form .hint {
...
+}
+.uikit-btn:focus,.uikit-btn:hover {
+  background-color: #135566 !important;
```
Comment (marcoalfonso) [inline public/custom.css]: 👎

**Comment by maxious** [comment]
now with less !important @marcoalfonso

**Pull request merged**